magento/pwa-studio
# [Doc] css modules tutorial

## Description

This PR revises and publishes the css modules tutorial contributed by @rossmc 

## Related Issue

Closes PWA-304

## Acceptance 

any developer

### Verification Stakeholders

any developer

### Specification



### Verification Steps

1. Navigate to the pwa-devdocs directory: `cd pwa-devdocs`
2. Run linting on the new topic: `yarn lint src/tutorials/pwa-studio-fundamentals/css-modules/index.md`
3. Verify no linting issues
4. Start HTML preview server: `yarn develop`
5. Navigate to the tutorials section
6. Verify left nav entry for the new topic and clicking on it takes you to the correct page

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
